infra/aws/terraform/modules/registry-k8s-io-s3-bucket/main.tf

```diff
@@ -25,11 +27,25 @@ module "s3_bucket" {
   source  = "terraform-aws-modules/s3-bucket/aws"
   version = "3.15.1"
 
-  bucket = base64sha256(local.bucket_name)
-  acl    = "public-read"
+  bucket = replace(lower(base64sha256(local.bucket_name)), "/[^a-zA-Z0-9-_]/", "")
```

Comment:
One reason of doing this is to ensure we respect bucket naming rules: https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html

Reply:
I completely forgot about his input. 🤦🏾 

We don't have detailed requirements for naming. The idea was to completely obfuscate the bucket name. I think we can revisit the `bucket_name`. I don't have a strong opinion about this.

Reply:
I think its ok either way - if you go ahead with what you have currently, I would just recommend adding some descriptive tags so that you can connect the buckets back to the repo/use case, etc., since the name will be obfuscated

Reply:
I'll add tags as a follow-up PR so we can have a dedicated conversation.